streamer/pulp/streamer/server.py

```diff
@@ -17,6 +17,19 @@
 
 logger = logging.getLogger(__name__)
 
+# These HTTP/1.1 headers are defined as being hop-by-hop and should
+# not be passed back to clients. All other headers defined by
+# HTTP/1.1 are end-to-end headers and should be passed back to the
+# client. See RFC 2068, section 13.5.1 for more information.
+HOP_BY_HOP_HEADERS = [
+    'connection',
+    'keep-alive',
+    'public',
+    'proxy-authenticate',
+    'transfer-encoding',
+    'upgrade',
+]
```

Comment:
nit: These are all lowercase for a reason. That reason should probably be in the comment above this list definition.